src/renderer/services/file-storage-service/index.ts

```diff
@@ -11,6 +11,10 @@ import { HttpClient } from "../types";
 export interface FSSUpload extends JSSJob {
   serviceFields: {
     fileId?: string;
+    pre_upload_md5?: string;
```

Comment:
Sort of following up on another comment I made in App/index.tsx, if this is a string would it ever equal an integer like file size?

Reply:
hm, that should be a number.